pkg/engine/customscripts.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+package engine
```

Comment:
Are these functions a re-organization of existing functions?

Reply:
yes